source/mainnet/net/references/grpc.rst

```diff
@@ -0,0 +1,886 @@
+.. _grpc-documentation:
+
+====================
+ gRPC documentation
+====================
+
+The Concordium node has a `gRPC <https://grpc.io/>`_ interface that enables
+queries, sending of transactions, and more.
+While the gRPC interface is powerful, it is not the most convenient tool.
+We provide a number of SDKs that build on top of the gRPC interface, which are
+much more ergonomic to use.
+
+These are the SDKs we currently provide:
+
+- `Concordium Rust SDK <https://github.com/Concordium/concordium-rust-sdk>`_
+- `Concordium Javascript (Node / Web) SDK <https://github.com/Concordium/concordium-node-sdk-js>`_
+- `Concordium Java SDK <https://github.com/Concordium/concordium-java-sdk>`_
+- `Concordium .NET (C#) SDK <https://github.com/Concordium/concordium-net-sdk>`_
+- `Concordium Go SDK <https://github.com/Concordium/concordium-go-sdk>`_
+
+The gRPC interface uses a mixture of protobuf defined types and JSON. For the
+JSON types, a `JSON Schema <https://json-schema.org/>`_ is provided.
+
+For the JSON schemas, you may find these two tools useful:
+
+- `JSON Schema Faker <https://json-schema-faker.js.org/>`_: Generates fake data
+  that conforms to the provided schema.
+- `JSON Schema Validator <https://www.jsonschemavalidator.net/>`_: Validates
+  whether the JSON input follows the provided schema.
+
+
+Notation
+========
+
+This page uses the following notation:
+
+- ``?a`` means ``a`` OR ``null``.
+
+  - JSON schemas for ``?a``-types, will only specify the ``a``. But the node may
+    also return ``null``.
+
+- ``[a]`` means a list of type ``a``.
+- JSON Schemas follow the `JSON Schema (Draft 7) <https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-01>`_.
+- Protobuf code follow the `Protocol Buffers Version 3 Language Specification <https://developers.google.com/protocol-buffers/docs/reference/proto3-spec>`_.
+- ``Foo ::= (n: UInt16) (bars: Byteⁿ)``
+
+  - Denotes the serialization of the type ``Foo``, which starts with an unsigned
+    16-bit Integer, ``n``, which describes the number of bytes used to represent
+    ``bars``. So if ``n == 100``, then ``bars`` is represented with ``100`` bytes.
+
+Transactions
+============
+
+.. function:: SendTransaction(networkId, payload) -> Bool
+
+   Send a transaction to the given network. See more information about the
+   payload in the :ref:`grpc-transaction-encoding` section.
+
+   :param Int32 networkId: The network that the transaction should be sent to (default is 100).
+   :param payload: Binary encoding of the transaction payload.
+   :type payload: |grpc-block-item|_
+   :returns: Whether the transaction succeeded.
+   :rtype: Bool
```

Comment:
I recommend stating in the description that the node will do basic transaction validation such as signature checks, nonce checks, and if these fail, the call will return a gRPC error.